website/content/docs/plugins/install-plugins.mdx

```diff
@@ -171,37 +200,32 @@ will avoid conflicting with other plugins for other tools, like Terraform.
 </Tab>
 <Tab heading="manually (multi-component plugin)">
 
--> The [`packer plugins`](/packer/docs/commands/plugins) command allows to install plugins without going through
-`init`. For manual installation of plugin binaries, without the `packer plugins` command, please continue reading.
+-> The [`packer plugins`](/packer/docs/commands/plugins), available from Packer v1.8.0, command allows 
+you to install plugins without going through `init`. 
 
-The easiest way to manually install a plugin is to name it correctly, then place
-it in the proper directory. To name a plugin correctly, make sure the binary is
-named `packer-plugin-NAME`. For example, `packer-plugin-amazon` for a "plugin"
-binary named "amazon". This binary will make one or more components available to
-use. Valid types for plugins are down this page.
-
-Once the plugin is named properly, Packer automatically discovers plugins in
-the following directories in the given order. If a conflicting plugin is found
-later, it will take precedence over one found earlier.
+```shell
+packer plugins install github.com/hashicorp/vagrant
+```
 
-1.  The directory where `packer` is, or the executable directory.
+## Plugin Installation Workflow
+Plugin installation via `packer plugins install` works similar to that of the `packer init` command, with the following
+exceptions no `required_plugins` block required and can be used with both legacy JSON and HCL2 templates.
 
-2.  The `$HOME/.packer.d/plugins` directory, if `$HOME` is defined (Unix)
+* [`packer plugins install`](/packer/docs/commands/plugins) will install plugins in the **last** directory
+in the following numbered list.
 
-3.  The `%APPDATA%/packer.d/plugins` if `%APPDATA%` is defined (Windows)
+1. `PACKER_PLUGINS_PATH` if set will be the sole location for installing plugins. All other 
+plugin directories will be ignored. 
+1. `PACKER_CONFIG_DIR`\plugins on Windows systems, or `PACKER_CONFIG_DIR`/plugins on all other systems.
```

Comment:
Almost commented that this should be a `2` instead, but it seems that it renders correctly in the final docs, that's good to know.